src/SEnergyCorrelator.h

```diff
@@ -128,6 +129,9 @@ namespace SColdQcdCorrelatorAnalysis {
       vector<TH1D*> m_outHistVarLnDrAxis;
       vector<TH1D*> m_outHistErrLnDrAxis;
 
+      vector<TH1D*> weight_outHistErrDrAxis;
+      vector<TH1D*> Tweight_outHistErrDrAxis;
```

Comment:
A common practice is to indicate what variables are formal members of the class by prefixing their name with `m_`. I'd suggest the following changes to follow that practice and make the names consistent between the two different calculations.

(p.s. this might require further changes in light the above comment about what weights to use.)

```suggestion
      vector<TH1D*> m_outPackageHistVarLnDrAxis;
      vector<TH1D*> m_outPackageHistErrLnDrAxis;

      vector<TH1D*> m_outManualHistErrDrAxis_weightEne;
      vector<TH1D*> m_outManualoutHistErrDrAxis_weightEt;
```